test/node-next/es2018/cursor_async_iterator.test.js

```diff
@@ -76,12 +77,19 @@ describe('Cursor Async Iterator Tests', function () {
     metadata: { requires: { node: '>=10.5.0' } },
     test: async function () {
       const cursor = collection.find();
+      const cursorStream = cursor.stream().pipe(
```

Comment:
@mbroadst Unfortunately changing the code made it so I can't reply directly to our old thread about this. I've modified the test so that `cursor.close()` gets called before the first object is returned from the stream, which I think was the intent of the test as originally written. It works this way; do you think that's good enough, or do we want to dive further into why for stream gets consumed entirely before the first iteration of the loop's body gets executed?